platform/machine/qemu/flight.go

```diff
@@ -70,37 +65,6 @@ func NewFlight(opts *Options) (platform.Flight, error) {
 		diskImagePath: opts.DiskImage,
 	}
 
-	if opts.Distribution != "cl" {
-		// don't apply CL-specific mangling
-		opts.UseVanillaImage = true
-	}
-	if !opts.UseVanillaImage {
-		info, err := util.GetImageInfo(opts.DiskImage)
-		if err != nil {
-			qf.Destroy()
-			return nil, err
-		}
-		if info.Format != "raw" {
-			// platform.MakeCLDiskTemplate() needs to be able to mount
-			// partitions
-			plog.Debug("disk image is in qcow format; not enabling console logging")
-			opts.UseVanillaImage = true
-		}
-	}
-	if !opts.UseVanillaImage {
-		plog.Debug("enabling console logging in base disk")
-		qf.diskImageFile, err = platform.MakeCLDiskTemplate(opts.DiskImage)
```

Comment:
This is the only assignment to `diskImageFile`, so that member can be dropped.  It looks like it was copypasted into `unprivqemu`, also.

Reply:
All gone! :recycle: